src/node/test/channels.cpp

```diff
@@ -41,6 +41,8 @@ using MsgType = std::array<uint8_t, msg_size>;
 static NodeId self = std::string("self");
 static NodeId peer = std::string("peer");
 
+static constexpr auto default_curve = crypto::CurveID::SECP384R1;
```

Comment:
There's also `crypto::service_identity_curve_choice` as the default choice, maybe we should renamed that though.

Reply:
Yes - I'd like to remove that value entirely, since its now badly named. But doing so involves touching a bunch of unit tests with this kind of change, so I'll save it for a separate PR.

Reply:
Sure, sounds good!